ClassicLootManager/Modules/Auctioning/Common/AuctionInfo.lua

```diff
@@ -236,8 +236,9 @@ end
 
 local function AddItemInternal(self, item)
     if self.items[item:GetItemID()] then
-        LOG:Message("%s already in auction list.", item:GetItemLink())
```

Comment:
Keep this line but add information that we are adding new:
`LOG:Message("%s already in auction list. Increasing the amount to award.", item:GetItemLink())`